### PR TITLE
Improves documentation and fixes small bug

### PR DESCRIPTION
**Pull request by marlycormar**
### Overview:
- Targets issue #6.
- Fixes bug:  if the descriptive text was left blank, the word *null* would show up on the token request page.

(Thanks @tlstoffs for making corrections to the workflow section on the `README.md`)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,3 +36,12 @@ the rewritten target URL might look like this:
 
 and the text 'About the REDCap Mobile App' will be added on the 'REDCap Mobile App' before the title 'What is the REDCap Mobile App?'.
 
+
+## Module Workflow
+1. The user requests a token: On the project in question, the user clicks on the left menu link **REDCap Mobile App** and then on the button **Request API token**. He is redirected to the URL provided during the configuration of the module. Once he completes the questionnaire, the token request will be submitted.
+2. The project manager/administrator receives the request.
+3. The project manager/administrator approves the request.
+4. The project manager/administrator disables the module: The project manager/administrator disables the module in the project by going to the ** External Modules** page (under Applications) and clicking **Disable** next to the **Route Mobile Api Token Requests to Survey** module.
```
Comment (pbchase) [inline README.md]: we must be clear about where the module disable event occurs.  Maybe the title for this bullet should read `The project manager/administrator disables the module at the project level`.  Then add emphasis in the sentence.  Maybe like this:

```
The project manager/administrator disables the module level: The project manager/administrator disables the module in the _project_ by going to the **External Modules** page (under Applications) and clicking **Disable** next to the **Route Mobile Api Token Requests to Survey** module.
```

File: README.md [hunk 2]
```diff
@@ -36,3 +36,12 @@ the rewritten target URL might look like this:
...
+5. The user goes back to the project in question and clicks the **Request API token** button on the **REDCap Mobile App** page.
```
Comment (pbchase) [inline README.md]: I think step 5 needs to be preceded by a new step that reads something like this

```
The project manager/administrator contacts the user: The project manager/administrator sends a message to the user asking him to revisit the mobile API token request page to complete the token request.
```

**Review by pbchase — CHANGES_REQUESTED** [review]
The README could use just a few more changes. See my notes.

**Comment by marlycormar** [comment]
@pbchase, thanks for the input. I made improvements in the README.md based on your comments. Could you please review them?

**Comment by pbchase** [comment]
Looks good.  I'll merge!

**Pull request merged**